airflow/providers/amazon/aws/utils/emailer.py

```diff
@@ -32,12 +35,23 @@ def send_email(
     mime_subtype: str = 'mixed',
     mime_charset: str = 'utf-8',
     conn_id: str = 'aws_default',
+    from_address: str = None,
+    from_name: str = None,
     **kwargs,
 ) -> None:
     """Email backend for SES."""
     hook = SESHook(aws_conn_id=conn_id)
+    if from_name and from_address:
+        mail_from=f'{from_name} <{from_address}>'
+    elif from_address:
+        mail_from = from_address
+    else:
+        raise AirflowConfigException("mail_from must be provided")
+
```

Comment:
Should we do same as https://github.com/apache/airflow/blob/8ae2bb9bfa8cfd62a8ae5f6edabce47800ccb140/airflow/providers/sendgrid/utils/emailer.py#L70-L74 instead of `from_address` and `from_name`?